GoogleForCreators/web-stories-wp
# Added ability to copy-paste background elements

## Summary

This modifies both how elements are copied (background shapes are added page background color), and how elements are pasted (pasted background elements are filtered out and handled separately).

## Relevant Technical Choices



## To-do

- [x] Modify copying/cutting background shapes (add page color to element)
- [x] Modify how background elements are handled on paste (separate handling)
- [x] Wait for both dependent PR's to be merged (#1925, ~#2158~)
- [x] Update and amend unit tests for copy-paste
- [x] Add karma test
- [x] Also handle duplication
- [x] Add karma test for duplication


## User-facing changes

Copy-pasting elements now does not allow a user to make multiple copies of a background element (shape or media) - if you paste a background element, it replaces what was already background on the given page - thus completely removing what was there.

If you copy and paste a bg to the same page, nothing happens (actually the bg element is updated, but to something identical to itself).

## Testing Instructions

1. Select the background element on any page (shape or image)
1. Copy the element via keyboard shortcut
1. Paste via keyboard shortcut
1. Observe that nothing happens.

For full QA testing instructions, please see #1440.

## Fixes

Fixes #1440
Fixes #2709
